anime-list-master.xml

```diff
@@ -2352,7 +2352,7 @@
       <mapping anidbseason="0" tvdbseason="0">;1-0;</mapping>
     </mapping-list>
   </anime>
-  <anime anidbid="566" tvdbid="unknown" defaulttvdbseason="3" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="566" tvdbid="349691" defaulttvdbseason="3" episodeoffset="" tmdbid="" imdbid="">
```

Comment:
There is no Season 3

Reply:
Thanks, this is meant to be specials this should be the correct way for it?

Reply:
Season 0 is TVDB Specials, that's correct. Just double-check the need for episode offsets
This doesn't need it, first 6 TVDB specials match the AniDB entry's episodes